contracts/drops/HolographDropCreator.sol

```diff
@@ -77,7 +77,7 @@ contract HolographDropCreator is Initializable {
     HolographERC721Drop impl = HolographERC721Drop(payable(implementation));
     HolographERC721DropProxy erc721DropProxy = new HolographERC721DropProxy();
     DropInitializer memory initialzer = DropInitializer(
-      impl.holographFeeManager.address,
+      address(0),
```

Comment:
Oh I think we want to still have the fee manager in order to charge the 5% fee that goes to Holograph

Reply:
This is going to be moved to the HolographRoyalty contract